Stripe/STPBankAccount.m

```diff
@@ -127,7 +124,7 @@ + (NSArray *)requiredFields {
              ];
 }
 
-+ (instancetype)decodedObjectFromAPIResponse:(NSDictionary *)response {
++ (nullable instancetype)decodedObjectFromAPIResponse:(nullable NSDictionary *)response {
```

Comment:
Oh yeah, just double checked. I was looking at the property definitions in the implementation. 

I think the only one out of sync is `fingerprint` is `nullable` in the header and `nonnull (default)` in the implementation.

Also the `accountHolderType` will default to `STPBankAccountHolderTypeIndividual` if it's not defined in the response which is a little sketchy but probably the right fallback anyway.